rclcpp/minimal_action_client/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ find_package(example_interfaces REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_action REQUIRED)
 
+add_executable(action_client_member_function member_function.cpp)
+ament_target_dependencies(action_client_member_function
+  "rclcpp"
+  "rclcpp_action"
+  "example_interfaces")
+
 add_executable(action_client_not_composable not_composable.cpp)
```

Comment:
I think this also a good opportunity to remove the non_composable variants.

Reply:
My understanding is that this repo contains minimal examples for all variants, and those that are the recommended way to do things should be in [demos](https://github.com/ros2/demos) or [tutorials](https://github.com/ros2/tutorials) ([relevant ROS Answer](https://answers.ros.org/question/318089/difference-of-ros2-demos-and-ros2-examples/)). Although, it does seem a little redundant IMO.

Reply:
Fair enough.  I think that this may warrant a discussion at the next ROS2 meeting, because I was under the impression that the non-composable stuff can be dropped.

Reply:
I've added it to the agenda.